olive/model/handler/onnx.py

```diff
@@ -64,9 +69,18 @@ def model_path(self) -> str:
         model_path = super().model_path
         return get_onnx_file_path(model_path, self.onnx_file_name) if model_path else None
 
-    def load_model(self, rank: int = None) -> ModelProto:
+    def load_model(self, rank: int = None, enable_fast_mode: bool = False) -> ModelProto:
```

Comment:
We can't add
```
if self.model is not None:
    return self.model
self.model = onnx.load(self.model_path)
return self.model
```
here like how PyTorchModelHandler. Some passes may reuse the same modelhandler for search and we don't want to reuse the same loaded model but need to load model again from new model path. It would be great if someone knows which pass is doing such search and we can see if we can update it.